app/sync.js

```diff
@@ -22,6 +22,16 @@ const settings = require('../js/constants/settings')
 const CATEGORY_MAP = syncUtil.CATEGORY_MAP
 const CATEGORY_NAMES = Object.keys(categories)
 
+const SYNC_ACTIONS = Object.values(syncConstants)
+const SYNC_ACTIONS_WITH_ITEMS = [
+  syncConstants.SYNC_ADD_SITE,
+  syncConstants.SYNC_UPDATE_SITE,
+  syncConstants.SYNC_REMOVE_SITE,
+  syncConstants.SYNC_ADD_SITE_SETTING,
+  syncConstants.SYNC_UPDATE_SITE_SETTING,
+  syncConstants.SYNC_REMOVE_SITE_SETTING
+]
```

Comment:
i think these constants should go inside validateAction since they are only used in there